deploy/deploy-wmagent-venv.sh

```diff
@@ -0,0 +1,628 @@
+#!/usr/bin/env bash
+
+help(){
+    echo -e $*
+    cat <<EOF
+    Usage: wmagent-venv-deploy.sh [-s] [-n] [-v] [-y]
```

Comment:
these examples/documentation does not match the script name

Reply:
done